exchanges/binance/binance.go

```diff
@@ -178,10 +176,15 @@ func (b *Binance) GetAggregatedTrades(symbol string, limit int) ([]AggregatedTra
 
 	params := url.Values{}
 	params.Set("symbol", strings.ToUpper(symbol))
-	params.Set("limit", strconv.Itoa(limit))
+	if limit > 0 {
+		params.Set("limit", strconv.Itoa(limit))
+	}
 
-	path := fmt.Sprintf("%s%s?%s", b.API.Endpoints.URL, aggregatedTrades, params.Encode())
+	if end.Sub(start) > time.Hour {
+		return resp, errors.New("start and end cannot be greater than 1 hour")
+	}
 
+	path := b.API.Endpoints.URL + aggregatedTrades + "?" + params.Encode()
```

Comment:
Just as a FYI, as common is already imported, can use path := common.EncodeURLValues(b.API.Endpoints.URL+aggregatedTrades, params) if you find it more convenient